package.json

```diff
@@ -54,7 +54,7 @@
     "bytes-counter": "~1.0.0",
     "commander": "~2.9.0",
     "csv-parser": "~1.9.3",
-    "hapi": "~11.1.3",
+    "hapi": "~16.0.0",
```

Comment:
The following entry should be added to CHANGES_NEXT_RELEASE file:

```
Upgrade hapi to 16.0.0
```

Reply:
Thanks. I've added this entry to CHANGES_NEXT_RELEASE file.

Reply:
Fixed in eba21e257d6fde376e618deba5b4a22d66109bbc